Enhance post-processing module performance (#865)
lethe-cfd/lethe: doc/source/examples/unresolved-cfd-dem/liquid-solid-fluidized-bed/liquid-solid-fluidized-bed.rst

.. important::

>     This example includes a post-processing file written in Python and using `lethe_pyvista_tools <../../../tools/postprocessing/postprocessing.html>`_. module.

Reviewer: ```suggestion
    This example includes a post-processing file written in Python that uses the `lethe_pyvista_tools <../../../tools/postprocessing/postprocessing.html>`_. module.
```
